numba/cuda/tests/cudapy/test_transpose.py

```diff
@@ -0,0 +1,75 @@
+from __future__ import print_function, division, absolute_import
+import numpy as np
+import math
+from numba import cuda, float32, float64, complex64, complex128
+from numba.cuda.testing import unittest
+from numba.cuda.cudadrv.driver import driver
+from ddt import ddt, data, file_data, unpack
+
+
+# There ought to be a better way to do this !
```

Comment:
There is `numba.numpy_support.from_dtype(dtype)` which will return the corresponding numba type for a given numpy type.